package.json

```diff
@@ -27,8 +27,15 @@
     "amqp-coffee": "0.1.30",
     "async": "2.1.5",
     "debug": "2.6.3",
-    "lodash": "4.17.4",
+    "lodash.clone": "^4.5.0",
     "lodash.get": "4.4.2",
+    "lodash.isfunction": "^3.0.8",
```

Comment:
I don't know what lodash.core contains, I can't find that on npmjs. 
Given that we agree on remove some functions, what we need will be less than now, So I guess core would be more than needed.